### ISSUE TITLE
Add reproducibility section 

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/intro_to_r#55]
I'm currently working on: https://github.com/jhudsl/Reproducibility_in_Cancer_Informatics 
But I can make a version for this audience/format/time slot. 

Will have to determine what the priorities are.

**Comment by carriewright11** [comment]
Sounds good. I kinda think this should occur on the first day to introduce R Markdown, after we introduce RStudio. What do you think @avahoffman and @cansavvy ?

**Comment by avahoffman** [comment]
I think early is best :)

**Comment by cansavvy** [comment]
Yes early but also integrated and dispersed throughout the course. So when we do something for the sake of reproducibility, we point it out.

**Comment by carriewright11** [comment]
We can work on this together perhaps to reduce the burden for @cansavvy

**Comment by cansavvy** [comment]
I think it should just involve reformatting material already written in https://github.com/jhudsl/Reproducibility_in_Cancer_Informatics and https://github.com/jhudsl/Adv_Reproducibility_in_Cancer_Informatics but maybe you guys can point out what topics should be emphasized. 

I think previously @carriewright11 mentioned the [Notebooks chapter](https://jhudatascience.org/Reproducibility_in_Cancer_Informatics/using-notebooks.html) for sure, but maybe also the [durable code chapter](https://jhudatascience.org/Reproducibility_in_Cancer_Informatics/writing-durable-code.html) ?

**Comment by carriewright11** [comment]
I agree.. yes, there is some brief material about chunk settings

**Comment by carriewright11** [comment]
I lost where it is though  :P

**Comment by carriewright11** [comment]
> I lost where it is though :P

here: https://github.com/jhudsl/intro_to_r/blob/gh-pages/Reproducibility/Reproducibility.Rmd
